src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -319,6 +319,9 @@ public static final class Facts {
     @Fetch(FetchMode.SELECT)
     private Map<String, String> environmentIds;
 
+    @Column(name = "reg_auth_method")
+    private String registrationAuthenticationMethod;
```

Comment:
You an use Hibernate's `@Enumerated(EnumType.STRING)` annotation here, and make the field type be the enum class iself